nifi-nar-bundles/nifi-rethinkdb-bundle/nifi-rethinkdb-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,30 @@
+nifi-rethinkdb-nar
+Copyright 2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+===========================================
+Apache Software License v2
+===========================================
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) RethinkDB Java Driver
+    The following NOTICE information applies:
+    RethinkDB Language Drivers
+
+    Copyright 2010-2012 RethinkDB
+
```

Comment:
lines 18-29 can be removed. Also need to add 'commons-io' in here.  Can copy it from the top level NOTICE that has it.